fitbenchmarking/fitbenchmarking
# add factor of 2 in base jacobian

#### Description of Work

Fixes #635 


#### Testing Instructions

1. check scipy/mantid jacobians work
2. check docs are up to date
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
